crypto/core_namemap.c

```diff
@@ -108,7 +108,7 @@ int ossl_namemap_empty(OSSL_NAMEMAP *namemap)
         return 1;
 
     if (!CRYPTO_THREAD_read_lock(namemap->lock))
-        return -1;
+        return 0;
```

Comment:
Isn't this contrary to what you documented and it's use in `ossl_namemap_stored()`?

Reply:
Arg.  Missed one.  Updated commit pushed.